torchmetrics/functional/nlp.py

```diff
@@ -69,49 +48,28 @@ def bleu_score(
         >>> reference_corpus = [['there is a cat on the mat'.split(), 'a cat is on the mat'.split()]]
         >>> bleu_score(translate_corpus, reference_corpus)
         tensor(0.7598)
+
```

Comment:
can you include a warning similar to this https://github.com/PyTorchLightning/metrics/blob/a4d2a7a81ba482f28edd30bd76abc3cffbaca548/torchmetrics/functional/regression/mean_relative_error.py#L43-L44 in the docstring